src/ir_passes/await.ml

```diff
@@ -12,6 +12,8 @@ let fresh_cont typ ans_typ = fresh_var "k" (contT typ ans_typ)
 
 let fresh_err_cont ans_typ  = fresh_var "r" (err_contT ans_typ)
 
+let fresh_bail_cont ans_typ = fresh_var "b" bail_contT
```

Comment:
I _think_ I know what I dislike about the `finally_cont`. It is because the `finally` blocks are composed into all sorts of continuations, but what is meant here is a chain of `finally`-blocks _only_, which is invoked in the specific situation of a trapping `await` continuation. That's why I came up with _bail_, because we are leaving the current context doing only the minimal amount of work possible.

Reply:
Of course we can come up with a clean nomenclature any time we wish. Not sure if this naming thing warrants a delay in merging.

Reply:
yeah, ok. My only real preference for finally is that it would show up in a grep for finally

Reply:
happy to live with it.